src/main/java/uk/gov/hmcts/reform/prl/services/ManageOrderEmailService.java

```diff
@@ -106,6 +111,42 @@ public void sendEmailToApplicantAndRespondent(CaseDetails caseDetails) {
 
     }
 
+    public void sendFinalOrderIssuedNotification(CaseDetails caseDetails) {
+        CaseData caseData = emailService.getCaseData(caseDetails);
+        if (caseData.getState() == State.ALL_FINAL_ORDERS_ISSUED) {
+            sendNotificationToRespondentSolicitor(caseDetails);
+            sendNotificationToRespondent(caseDetails);
+        }
+    }
+
+    private void sendNotificationToRespondent(CaseDetails caseDetails) {
+        CaseData caseData = emailService.getCaseData(caseDetails);
+
+        for (Element<PartyDetails> respondent : caseData.getRespondents()) {
+            emailService.send(
+                respondent.getValue().getEmail(),
+                EmailTemplateNames.CA_CITIZEN_RES_NOTIFICATION,
+                buildRespondentEmail(caseDetails, respondent.getValue()),
+                LanguagePreference.english
+            );
+        }
+    }
+
+    private void sendNotificationToRespondentSolicitor(CaseDetails caseDetails) {
+
+        for (Map<String, List<String>> resSols : getRespondentSolicitor(caseDetails)) {
```

Comment:
If there are multiple respondent solicitors?

Reply:
we are getting list of solicitor inside map,so I believe we will get all respondent solictors information